pycaret/internal/preprocess/time_series/forecasting/preprocessor.py

```diff
@@ -112,6 +114,101 @@ def _add_imputation_steps(
         else:
             self.pipe_steps_exogenous.extend([("numerical_imputer", num_estimator)])
 
+    def _limitation(self,
```

Comment:
Could you rename this to _limiter? _limitation sounds like some limitation of the module itself. Sorry just being picky :)